### ISSUE TITLE
All styles are reported as unused in exported default function.

### ISSUE THREAD

**Issue by joarkosberg** [RodSarhan/eslint-plugin-react-native-unistyles#4]
Thanks for the nice eslint plugin 😊 

While testing I saw it at the moment reports all styles as unused in exported default functions.

### Example

Using styles in exported default function does report all as unused
```
export default function Something
```

Using styles in exported function does work as expected
```
export function Something
```

**Comment by RodSarhan** [comment]
@joarkosberg good catch, thanks! 
I'll release a fix for that soon.

**Comment by RodSarhan** [comment]
@joarkosberg just to make sure we're on the same page, can you please add a simple code snipper of both cases?

**Comment by joarkosberg** [comment]
@RodSarhan Sorry for a bit late reply, but it works with latest version, thanks! 

Here is simple code snippet if still needed.
With default export
```
import React from 'react';
import { View } from 'react-native';
import { createStyleSheet, useStyles } from 'react-native-unistyles';

export default function Test() {
    const { styles } = useStyles(styleSheet);

    return <View style={styles.modal}></View>;
}

const styleSheet = createStyleSheet(() => ({
    modal: {
        margin: 0,
    },
}));
```


Without default export
```
import React from 'react';
import { View } from 'react-native';
import { createStyleSheet, useStyles } from 'react-native-unistyles';

export function Test() {
    const { styles } = useStyles(styleSheet);

    return <View style={styles.modal}></View>;
}

const styleSheet = createStyleSheet(() => ({
    modal: {
        margin: 0,
    },
}));
```

**Comment by RodSarhan** [comment]
No worries i just wanted to confirm, thanks for reporting
